### PR TITLE
Laravel 6.0

### PR DESCRIPTION
**Pull request by AnsellC**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -72,5 +72,5 @@
         }
     },
     "minimum-stability": "dev",
-    "prefer-stable" : true
-}
```
Comment (antonkomarev) [inline composer.json]: Fix missing line ending, please.

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@ vendor/
 composer.lock
 composer.phar
 phpunit.xml
+.phpunit.result.cache
```
Comment (antonkomarev) [inline .gitignore]: 1. Reorder `.phpunit.result.cache` alphabetically.
2. Fix missing line ending

**Comment by antonkomarev** [comment]
Thank you for the contribution @AnsellC. It's merged. Release will be published tomorrow.

**Comment by AnsellC** [comment]
Thank you @antonkomarev looking forward to it!

**Comment by antonkomarev** [comment]
[Laravel Ban v4.1 has been released](https://github.com/cybercog/laravel-ban/releases/tag/4.1.0).

**Pull request merged**